Main.cpp

```diff
@@ -191,43 +239,63 @@ int main() {
       // canBus.printRxAll();
     }
 
-    chThdSleepMilliseconds(50);
+    chThdSleepMilliseconds(10000);
```

Comment:
Exactly my intention (as to scheduling and resource contention) and agreed on the 24 hours